shared/statestore/store.go

```diff
@@ -0,0 +1,164 @@
+package statestore
+
+import (
+	"bytes"
```

Comment:
I think this may indicate a need to rebase again it's not part of go-fil-components any more.